backend/substrapp/ledger_utils.py

```diff
@@ -154,6 +154,23 @@ def get_hfc():
         loop.close()
 
 
+def _get_signing_peers(strategy, current_peer, all_peers):
+    if strategy == 'SELF':
+        return [current_peer]
+    if strategy == 'MAJORITY':
```

Comment:
This looks more like `ALL `  than `MAJORITY`.
All Majority combinations will be constructed by   
```python
from itertools import combinations 
all_comb = combinations(all_peers, len(all_peers)//2 + 1) 

```

Reply:
Thank's @Kelvin-M, fixed here 3a9d353